### DISCUSSION TITLE
biome format - how is a list of paths specified?

### DISCUSSION THREAD

**Discussion by msssk** [biomejs/biome#2752] [Q&A]
The documentation for [`format`](https://biomejs.dev/reference/cli/#biome-format) says that PATH can be:
> Single file, single path or list of paths

What is the syntax to specify a list of paths? It would be good to provide an example in the documentation.

**Comment by ematipico** [comment]
An example: 

```shell
biome format --write ./src/pages ./src/app
```

Would you like to send a PR and update the formatter page?

**Comment by msssk** [comment]
Thank you, much appreciated. That is indeed simple and expected syntax; I am left wondering what was the actual problem I ran into the other day. When I tried to run `format` with multiple paths it failed with an error (something like `couldn't understand input`). If I ran it with a single path it worked. Today it works with a single path or multiple, so apparently the problem was not related to me not knowing multiple path syntax, but something else.

I do enjoy seeing examples of code, and I think even something as simple as [this](https://github.com/msssk/biome/commit/ff391021157a592ad5b74b9d29f4d3ed4b3f1c8b) would be helpful, but as a non-Rust dev looking at [CONTRIBUTING.md](https://github.com/biomejs/biome/blob/main/CONTRIBUTING.md) I'm disinclined to install all the tooling to run the tests and update the snapshot.

If you think this would be a useful PR (without me running tests and updating the snapshot) and someone else can take it over to get it over the finish line I'd be happy to submit it.

**Reply by ematipico** [reply]
Then install only the tools needed for your PR: `cargo-insta`.

To run the tests, you can still use `cargo test`
